helpers/mathjax.js

```diff
@@ -61,6 +61,16 @@ class HtmlBlockMathRenderer {
 			elm.style.height = '0.5rem';
 		});
 
+		// WIRIS outputs bad syntax for empty sub/superscripts, but changing these on write
+		// can be risky. Instead, change them on render to minimize potential damage if this
+		// impacts legitimate usages.
+		if (context.replaceNoneSubSuperScripts) {
+			elem.querySelectorAll('math mmultiscripts > none').forEach(elm => {
+				const mrow = document.createElementNS('http://www.w3.org/1998/Math/MathML', 'mrow');
+				elm.replaceWith(mrow);
```

Comment:
In practice I don't think the `none` element can have children, at least not the way WIRIS uses it, so it should be safe to do a straight replacement without any child element copying.